<launch>
	<arg name="cmd_vel_topic" default="/husky/cmd_vel" />
	<arg name="odom_topic" default="odom" />

	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
		<rosparam file="$(find hratc2015_entry_template)/settings/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find hratc2015_entry_template)/settings/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find hratc2015_entry_template)/settings/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find hratc2015_entry_template)/settings/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find hratc2015_entry_template)/settings/base_local_planner_params.yaml" command="load" />

		<remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
   		<remap from="odom" to="$(arg odom_topic)"/>
  	</node>

</launch>